index.html

```diff
@@ -0,0 +1,193 @@
+            make a difference. All the funds which will be contributed will buy food items.
+          </p>
+          <h4 class="head-h4">LET'S CONNECT</h4>
+          <h4 class="head-h4 head-h4-strong" ><strong>FROM (MOND 8AM) ~ (SUN 17AM)</strong></h4>
+          <h4 class="head-h4 head-h4-strong">Lets fundraise</h4>
+        </section>
+     
+      <!-- Card section Mobile & Desktop -->
+      <section id="program" class="bg-dark">
+        <h4 class="text-center">Main Program</h4>
+        <div class="container pb-3">
+          <div class="row justify-content-md-center">
+            <div class="col-md-6 col-lg-3">
+              <div class="card follow-up">
+                <img src="./image/registration.webp" class="card-img-top bg-white" alt="icon">                
+                <div class="follow-up-body">
+                  <h5 class="follow-up-title">Registration</h5>
+                  <p class="card-text">All the guests will register for attendance</p>
+                </div>
+              </div>
+            </div>
+            <div class="col-md-6 col-lg-3">
+              <div class="card follow-up">
+                <img src="./image/food-items.webp" class="card-img-top bg-white" alt="icon">
+                <div class="follow-up-body">
+                  <h5 class="follow-up-title">Collection of food items</h5>
+                  <p class="card-text">All food items will be collected and stored awaiting distribution.</p>
+                </div>
+              </div>
+            </div>
+            <div class="col-md-6 col-lg-3">
+              <div class="card follow-up">
+                <img src="./image/sleep_pillow_blanket-256.webp" class="bg-white card-img-top" alt="icon">
+                <div class="follow-up-body">
+                  <h5 class="follow-up-title">Collection of non-food items</h5>
+                  <p class="card-text">All non-food items will be collected and stored awaiting distribution.</p>
+                </div>
+              </div>
+            </div>
+            <div class="col-md-6 col-lg-3">
+              <div class="card follow-up">
+                <img src="./image/money.webp" class="card-img-top bg-white" alt="icon">
+                <div class="follow-up-body">
+                  <h5 class="follow-up-title">Cash fundraising</h5>
+                  <p class="card-text">Cash, money transfer and cheque will be collected.</p>
+                </div>
+              </div>
+            </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <!--Featured Speakers Section -->
+
+      <section id="speaker-wrapper">
+        <h2 class="section-header text-center" id="speaker">Featured Speakers</h2>
+        <div class="guide_bar"></div>
+        <div class="container">
+            <div class="row p-2" id="featured-speakers">
+            </div>
+        </div>
+    </section>
```

Comment:
Please, modify the `featured speakers` section to match the design guideline.

Use a lighter color for the mini background behind the images of the speakers. Also, stick to the font colors and sizes. The text for the speaker's title is red, italicized. The fonts are some of the fonts are smaller than it appears in your design.

Check the annotation below for more information.

Your design:
![image](https://user-images.githubusercontent.com/20554583/192085617-e08eb98a-adb8-4d9d-8395-332ee4fef07a.png)

Design guideline:
![image](https://user-images.githubusercontent.com/20554583/192085630-04858a6b-0842-4ccd-90ac-6560e0405b8a.png)